CompanionApp/Companion/Companion/Questionnaire.xaml

```diff
@@ -0,0 +1,222 @@
+                                        HorizontalTextAlignment="Start"
+                                        Text="{ Binding AnswerText }"
+                                        FontSize="15"
+                                        WidthRequest="500"                                            
+                                        LineBreakMode="WordWrap"
+                                        HeightRequest="{Binding TextHeight}">
+                                    </Label>
+                                </Grid>
+                            </DataTemplate>
+                        </grial:Repeater.SelectedItemTemplate>
+                    </grial:Repeater>
+                </StackLayout>
+                <StackLayout
+                    Orientation="Vertical"
+                    Grid.Row="1">
+                    <!-- SELECTED QUESTION VALIDATION-->
+                    <Label
+						Text="Please answer each question."
+						VerticalTextAlignment="End"
+						HorizontalTextAlignment="Center"
+						TextColor="{ DynamicResource SplashBackgroundColor }"
+                        Style="{ StaticResource LabelBoldStyle }"
+                        IsVisible="{ Binding IsAnswerEachQuestionVisible}"
+						FontSize="14" 
+                        HeightRequest="40"/>
+                    <!--NEXT BUTTON-->
+                    <Button
+                    Style="{ StaticResource PrimaryActionButtonStyle}"
+					x:Name="NextButton"
+					Text="Next"
+					Command="{ Binding MoveNext }"
+                    IsVisible="{ Binding IsNextButtonVisible}"
+					Margin="30,0,30,30"
+					WidthRequest="{
+                        grial:OnOrientationDouble
+                            LandscapePhone=200,
+                            LandscapeTablet=400
+                    }"
+					HorizontalOptions="{
+                        grial:OnOrientationLayoutOptions
+                            PortraitPhone=Fill,
+                            LandscapePhone=Center,
+                            PortraitTablet=Fill,
+                            LandscapeTablet=Center
+                    }" />
+                    <!--SUBMIT BUTTON-->
+                    <Button
+                    Style="{ StaticResource PrimaryActionButtonStyle}"
+					x:Name="SubmitButton"
+					Text="Submit"
+					Command="{ Binding Submit }"
+                    IsVisible="{ Binding IsSubmitButtonVisible}"
+					Margin="30,0,30,30"
+					WidthRequest="{
+                        grial:OnOrientationDouble
+                            LandscapePhone=200,
+                            LandscapeTablet=400
+                    }"
+					HorizontalOptions="{
+                        grial:OnOrientationLayoutOptions
+                            PortraitPhone=Fill,
+                            LandscapePhone=Center,
+                            PortraitTablet=Fill,
+                            LandscapeTablet=Center
+                    }" />
```

Comment:
Is the whitespacing on the xaml files all over the place in visual studio or is that just how it compiles here?